### ISSUE TITLE
Add support for storing the videos you own offline

### ISSUE THREAD

**Issue by ekaris** [ValveSoftware/steam-for-linux#4523]
Add support for storing the videos you own offline and backup them like steam games.

**Comment by lavnart** [comment]
You don't "own" anything on Steam, it is mere subscriptions given to you. And videos are explictly named "Streaming Videos".

**Comment by ekaris** [comment]
You're wrong, and that can be proven in the European Courts if ever needed.

**Comment by ekaris** [comment]
Back to topic, netflix now lets users store videos and shows offline and watch them offiline.
